interop/ksp/test-processor/src/test/kotlin/com/squareup/kotlinpoet/ksp/test/processor/TestProcessorTest.kt

```diff
@@ -47,6 +49,15 @@ class TestProcessorTest(private val useKsp2: Boolean) {
   @JvmField
   val temporaryFolder: TemporaryFolder = TemporaryFolder()
 
+  @Before
+  fun skipOnJdk8() {
+    val javaVersion = System.getProperty("java.version") ?: return // Don't skip if version unknown.
+    assumeFalse(
+      "These tests currently fail on JDK 8 when useKsp2 == false",
+      javaVersion.startsWith("1.8") && !useKsp2,
+    )
+  }
```

Comment:
Turning off these tests for now as they fail with an "internal error". Seems okay since it's just on JDK 8.